packaging/mariner/azure-nvme-utils.spec.tmpl

```diff
@@ -20,16 +20,15 @@ Utility and udev rules to help identify Azure NVMe devices.
 %autosetup
 
 %build
-%cmake -DVERSION="%{version}-%{release}" .
-make %{?_smp_mflags}
+%cmake -DVERSION="%{version}-%{release}"
+%cmake_build
 
 %install
-make install DESTDIR=%{buildroot}
+%cmake_install
+
+%check
+%ctest
 
 %files
-/usr/sbin/azure-nvme-id
 /lib/udev/rules.d/80-azure-nvme.rules
-
-%changelog
-* Mon Feb 26 2024 Chris Patterson <cpatterson@microsoft.com> - 0.1.1-1
-- Initial package
+/usr/sbin/azure-nvme-id
```

Comment:
```suggestion
%{_sbindir}/azure-nvme-id
```
FWIW https://fedoraproject.org/wiki/Changes/Unify_bin_and_sbin is coming so you could get ahead of the game and use `%{_bindir}` instead.